bindata/network/multus/multus.yaml

```diff
@@ -116,29 +117,109 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: multus
-      initContainers:
-      - name: multus-binary-copy
+      containers:
+      - name: kube-multus
         image: {{.MultusImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: [ "/bin/bash", "-c", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh &&
+            exec /entrypoint.sh
+            --multus-conf-file=auto
+            --multus-autoconfig-dir=/host/var/run/multus/cni/net.d
+            --multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig
+{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
+            --readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf
+{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
+            --readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf
+{{- end}}
+            --cleanup-config-on-exit=true
+            --namespace-isolation=true
+            --multus-log-level=verbose
+            --cni-version=0.3.1
+            --additional-bin-dir=/opt/multus/bin
+            --skip-multus-binary-copy=true
```

Comment:
Not sure whether this was answered before, what's the difference between letting multus do the binary copy and /entrypoint/cnibincopy.sh above?  What does the custom script do that multus entrypoint does not do by default?

Reply:
The binary copy here is an upstream option to let you copy the binary, but... it's "dumb" compared to how we do it in OCP. In OpenShift we want to copy the binary so that it's matched with the underlying host o/s. Which is why we have the mounted `cnibincopy.sh` which has the smarts to do this in the same fashion for all of the CNI binaries that we copy.